src/package1/CustomArrayList.java

```diff
@@ -250,7 +250,10 @@ public ListIterator<String> listIterator(int index) {
 
     @Override
     public List<String> subList(int fromIndex, int toIndex) {
-        return null;
+        String[] ar = new String[toIndex - fromIndex + 1];
+        System.arraycopy(elements,
+                fromIndex, ar, 0, (toIndex - fromIndex + 1));
+        return Arrays.asList(ar);
```

Comment:
Arrays.asList() will not return instance of your CustomArrayList, it returns ArrayList. How do you think is it OK?

![image](https://user-images.githubusercontent.com/107478608/225575738-c75d0acc-982d-4858-81b0-076e2f0fe4a9.png)

Reply:
I did not pay attention to this because the output requires a list, and ArrayList fits this category. Corrected.